codebuild/requirements.txt

```diff
@@ -0,0 +1,4 @@
+boto3
```

Comment:
What is this file for?

Reply:
requirements to run create_project.py

Reply:
Since this is just a "note to self" type file, can you make that clear? Structure it more like a readme?